stories/tile.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Tile } from '../src/components'
+
+storiesOf('Tile', module)
+  .add('Default', () => (
+    <Tile
+      children='Default'
+      type='default'
+    />
+  ))
+  .add('Muted', () => (
+    <Tile
```

Comment:
I think we should let the user choose what to use as children, for instance from the doc:
```html
<div class="uk-tile uk-tile-secondary">
   <p class="uk-h4">Secondary</p>
</div>
```